app/dashboard/templates/base.html

```diff
@@ -79,6 +79,12 @@
                             <i class="fa fa-stethoscope"></i>&nbsp;Tests<small>&nbsp;<sup>&#946;</sup></small>
                         </a>
                     </li>
+                    <li id="li-li-kcidb">
+                        <a id="kcidb-l" href="https://staging.kernelci.org:3000" target="_blank"
+                            title="View the KCIDB results">
```

Comment:
I don't feel strongly about this, but the "Tests" button also adds "(beta)" to the text you get when you hover over the button.

```suggestion
                            title="View the KCIDB results (beta)">
```

Reply:
Done.